source/plugins/go/src/extension/extension.go

```diff
@@ -48,9 +48,27 @@ func (e *Extension) GetOutputStreamId(datatype string) string {
 	return e.datatypeStreamIdMap[datatype]
 }
 
+func (e *Extension) GetContainerLogV2Flag() bool {

```

Comment:
rename the function name to "IsContainerLogV2"

Reply:
changed